src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -208,11 +208,11 @@ bool DirectoryService::RunConsensusOnFinalBlockWhenDSPrimary() {
 
   // kill first ds leader (used for view change testing)
 
-  // if (m_consensusMyID == 0 && m_viewChangeCounter < 1) {
-  //   LOG_GENERAL(INFO, "I am killing/suspending myself to test view change");
-  //   // throw exception();
-  //   return false;
-  // }
+  /*if (m_consensusMyID == 0 && m_viewChangeCounter < 1) {
```

Comment:
Remove unused code

Reply:
This code is used to test VC
```suggestion
  /*if (m_consensusMyID == 0 && m_viewChangeCounter < 1) {
```